chart/templates/workers/worker-deployment.yaml

```diff
@@ -214,6 +214,9 @@ spec:
           {{- include "custom_airflow_environment" . | indent 10 }}
           {{- include "standard_airflow_environment" . | indent 10 }}
         {{- end }}
+{{- if .Values.workers.extraContainers }}
+{{- toYaml .Values.workers.extraContainers | nindent 8 }}
```

Comment:
⚠️ Doing this with Kube executor would break things. If you add an extra container that has a daemon process running, then even if the "Airflow" worker container has finished, the pod is still active as the daemon in the other container is still up, and the pod never leaves the running state.

We (Astro) have had to hack around this to support running Istio sidecars, but it was not a pretty solution.

Reply:
I agree. I added the README that this option only supports CeleryExecutor.
And worker-deployment is not created in KubernetesExecutor.